app/experimenter/nimbus-ui/src/lib/visualization/mocks.tsx

```diff
@@ -47,12 +47,11 @@ export const MOCK_METADATA = {
  * overrides are present.
  */
 export const MOCK_METADATA_EXTERNAL_CONFIG = {
-  // TODO: account for the rest of these, EXP-1628
-  end_date: null,
-  enrollment_period: null,
+  start_date: "2021-05-26",
+  end_date: "2021-06-06",
+  proposed_enrollment: 9,
```

Comment:
I actually have no idea where I got `enrollment_period` from my previous commit. 😆 [These](https://github.com/mozilla/experimenter/issues/6200#issuecomment-898685666) are the options we need to account for.

Reply:
Hmm, from [when I last looked at the metadata from jetstream](https://github.com/mozilla/experimenter/pull/6264#discussion_r693148062), `enrollment_period` *was* one of the properties included.